hpat/hiframes/pd_series_ext.py

```diff
@@ -981,18 +981,22 @@ def generic_expand_cumulative_series(self, args, kws):
     return signature(return_type, recvr=self.this)
 
 
-# replacing cumsum/cumprod since arraydecl.py definition uses types.Array
-for fname in ["cumsum", "cumprod"]:
+# replacing cumprod since arraydecl.py definition uses types.Array
+for fname in ["cumprod"]:
     install_array_method(fname, generic_expand_cumulative_series)
 
 # TODO: add itemsize, strides, etc. when removed from Pandas
 _not_series_array_attrs = ['flat', 'ctypes', 'itemset', 'reshape', 'sort', 'flatten']
 
+# Array attributes which overlap Series attributes
+_excluded_array_attrs = ['cumsum']
+
 # use ArrayAttribute for attributes not defined in SeriesAttribute
 for attr, func in numba.typing.arraydecl.ArrayAttribute.__dict__.items():
     if (attr.startswith('resolve_')
             and attr not in SeriesAttribute.__dict__
-            and attr not in _not_series_array_attrs):
+            and attr not in _not_series_array_attrs
+            and attr.replace('resolve_', '') not in _excluded_array_attrs):
```

Comment:
Could you show all function names registered by `setattr(SeriesAttribute, attr, func)`?

Reply:
`print(attr.replace('resolve_', ''))` printed
`dtype`, `itemsize`, `shape`, `strides`, `ndim`, `size`, `flat`, `ctypes`, `flags`, `T`, `real`, `imag`, `transpose`, `item`, `itemset`, `nonzero`, `reshape`, `sort`, `view`, `ravel`, `flatten`, `prod`, `sum`, `mean`, `var`, `std`, `argmin`, `argmax`